bluepyopt/ephys/parameters.py

```diff
@@ -206,19 +216,23 @@ def __init__(
             self.value_scaler = parameterscalers.NrnSegmentLinearScaler()
         self.value_scale_func = self.value_scaler.scale
 
-    def instantiate(self, sim=None, icell=None):
+    def instantiate(self, sim=None, icell=None, params=None):
         """Instantiate"""
         if self.value is None:
             raise Exception(
                 'NrnSectionParameter: impossible to instantiate parameter "%s"'
                 ' without value' %
                 self.name)
 
+        _values = {"value": self.value}
```

Comment:
Can you be more verbose about what's happening here. Also give the variable a more meaningful name.

Do we really need all this logic in the NrnSectionParameter? It seems to make the class overly complex. Does this logic only apply to NrnSectionParameter for some reason, not to other types of parameters?

Reply:
@darshanmandge do you remember why we implemented it this way?

Reply:
Not sure but could be related to when there are multiple `param_dependencies`. The `_values` could be values of those parameters.

Reply:
@alejoe91, as I said in the other comment we were using them for the Hallermann model but it is not use anymore.